trie/database.go

```diff
@@ -281,8 +282,9 @@ func expandNode(hash hashNode, n node) node {
 
 // Config defines all necessary options for database.
 type Config struct {
-	Cache     int  // Memory allowance (MB) to use for caching trie nodes in memory
-	Preimages bool // Flag whether the preimage of trie key is recorded
+	Cache          int    // Memory allowance (MB) to use for caching trie nodes in memory
+	Preimages      bool   // Flag whether the preimage of trie key is recorded
+	StatsNamespace string // Namespace to use for stats
```

Comment:
Can we add a comment to explain why we use a namespace in the config here whereas in most other places we rely on a global namespace?